mmv1/third_party/terraform/services/pubsub/resource_pubsub_subscription_test.go

```diff
@@ -238,6 +238,10 @@ func TestAccPubsubSubscriptionBigQuery_serviceAccount(t *testing.T) {
 	topic := fmt.Sprintf("tf-test-topic-%s", acctest.RandString(t, 10))
 	subscriptionShort := fmt.Sprintf("tf-test-sub-%s", acctest.RandString(t, 10))
 
+	if acctest.BootstrapPSARoles(t, "service-", "gcp-sa-pubsub", []string{"roles/editor", "roles/viewer"}) {
```

Comment:
looks like these two are project editor / viewer vs. the ones needed by the two generated ones. Not sure what's needed to get the tests to show up as affected tests?

There is still a service account that's created and has the same roles assigned in the test in 2 of the steps, so this step won't solve the issue for that case.... maybe a sleep still needs to be added for the case where the service account is created and has roles assigned in the test?

Reply:
Also, we could easily enough not define the resources when the service account isn't created... but there's an explicit `depends` later that I think would be harder to get rid of....